communications/blogs/turing_joins_binder_federation.md

```diff
@@ -92,13 +90,20 @@ This project is openly developed and any and all questions, comments and recomme
 
 Read more about _The Turing Way_ in this [Turing Impact Story](https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science).
 
-## About Project Jupyter
+## About Project Binder
 
-[Project Jupyter](https://jupyter.org) is a non-profit project releasing 100% open-source software to support interactive data science and scientific computing across all programming languages.
+[Binder](https://mybinder.readthedocs.io/en/latest/about.html) is a community within Project Jupyter and received the [ACM Software System Award in 2017](https://awards.acm.org/award_winners/perez_9039634) for language agnostic tools which have become "a de facto standard for data analysis in research, education, journalism, and industry".
```

Comment:
I think we should put a full stop between "Binder is a community within Jupyter" and "Received the ...". Right now it is super hard to know if Binder or Jupyter received the ACM award.

Reply:
I move it to the "About Jupyter" section since Jupyter won the award